ui/package.json

```diff
@@ -47,7 +47,8 @@
     "build": "gulp build",
     "build:library": "gulp build:library",
     "test": "react-app-rewired test",
-    "fmt": "prettier --write ."
+    "fmt": "prettier --write .",
+    "supportedBrowsers": "gulp supportedBrowsers"
```

Comment:
please add a verb for it to be aligned with others, like, `gen:browserlist`

Reply:
Good naming, will update it.